IndicoV2.IntegrationTests/Submissions/SubmissionClientTests.cs

```diff
@@ -22,7 +22,7 @@ public class SubmissionClientTests
         public async Task SetUp()
         {
             var container = new IndicoTestContainerBuilder().Build();
-            _submissionsClient = container.Resolve<ISubmissionsClient>();
+            _submissionsClient = (SubmissionsClient)container.Resolve<ISubmissionsClient>();
```

Comment:
Why do we need additional casting? Can't we just use `ISubmissionsClient`?

Reply:
less warnings (obsolete is on the interface method, not class method).